src/util/Constants.js

```diff
@@ -169,6 +169,8 @@ exports.Endpoints = {
         makeImageUrl(`${root}/app-icons/${clientID}/${hash}`, { size, format }),
       AppAsset: (clientID, hash, { format = 'webp', size } = {}) =>
         makeImageUrl(`${root}/app-assets/${clientID}/${hash}`, { size, format }),
+      StickerBanner: (bannerID, format = 'webp', size) =>
+        makeImageUrl(`${root}/app-assets/710982414301790216/store/${bannerID}`, { size, format }),
```

Comment:
What's with this oddly specific ID?

Reply:
all the sticker banners are on that "Stickers" application, its id doesn't seem to be received from the API so I assume that it's in the client code, I also included it in the docs pr

Reply:
What about third-party stickers, such as Among Us's? Do they use the same application?

Reply:
![image](https://user-images.githubusercontent.com/20760160/122364191-765c6780-cf51-11eb-842f-2526a29320e8.png)

Reply:
hm i just noticed that the name for this is misleading: I put StickerBanner but it's more accurately StickerPackBanner, I'll fix that

guild stickers seem to not have packs so this wouldn't apply for them

Reply:
. 　。　　　　•　 　ﾟ　　。 　　.　　　.　　　 　　
 .　　。　　 。　. 　.　　 。　 ඞ 。　 . • .
   @iCrawl was ejected. . . 　 。　. 　　。　　
　　　.　　　　　. ,　　　　.　 .　　 .

---

Are you able to test that just in case? @advaith1 

I need confirmation in this.

Reply:
I can confirm that guild stickers do not have `pack_id`, if that's what you mean?

Reply:
Clarified from Discord - my main concern was about whether that ID is confirmed to be always constant (documented in Discord API's docs or an answer from a Discord developer in DDevs), for which @advaith1 has asked and is waiting for a reply.

I think it's prudent to wait for said question to be answered before we land this, as the hardcoded ID can change and potentially break this method in the future.

Reply:
Ref: https://github.com/discord/discord-api-docs/pull/3128

Reply:
![image](https://user-images.githubusercontent.com/11778454/122484490-8ee87400-cf89-11eb-823c-570842751b16.jpeg)